lib/aggregation/reporting/src/index.js

```diff
@@ -1276,7 +1276,10 @@ const runCLI = () => {
     systemToken.start();
   }
 
-  reporting().listen();
+  const server = reporting().listen();
+  
+  if (process.env.CONNECTION_TIMEOUT)
+    server.setTimeout(process.env.CONNECTION_TIMEOUT);
```

Comment:
Instead of having this in only reporting we can add it to webapp module:
https://github.com/cloudfoundry-incubator/cf-abacus/blob/9c26ee6d9433ddb971adf9926a1b9ad95bed6aeb/lib/utils/webapp/src/index.js#L86

This will also allow us to test it with a simple webserver that has the connection timeout set to under a second and expect an error.